server/block/model/enchanting_table.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+)
+
+// EnchantingTable is a model used by enchanting tables.
+type EnchantingTable struct{}
+
+// BBox ...
+func (EnchantingTable) BBox(cube.Pos, *world.World) []cube.BBox {
+	return []cube.BBox{full.ExtendTowards(cube.FaceDown, 0.25)}
+}
+
+// FaceSolid ...
+func (EnchantingTable) FaceSolid(cube.Pos, cube.Face, *world.World) bool {
+	return false
```

Comment:
Not even the bottom face?

Reply:
Nope, water flows through the bottom and blocks like lanterns do not hang.

Reply:
Got it, thanks :+1: